server/master/src/main/java/org/apache/accumulo/master/FateServiceHandler.java

```diff
@@ -171,7 +171,12 @@ public String invalidMessage(String argument) {
         });
 
         String tableId = ClientServiceHandler.checkTableId(master.getInstance(), oldTableName, tableOp);
-        String namespaceId = Tables.getNamespaceId(master.getInstance(), tableId);
+        String namespaceId;
```

Comment:
Could put these 5 lines in a method.  Also, I am thinking it should use NOT_FOUND instead of NAMEPSACE_NOTFOUND.